src/sys/prctl.rs

```diff
@@ -213,3 +215,14 @@ pub fn set_thp_disable(flag: bool) -> Result<()> {
 pub fn get_thp_disable() -> Result<bool> {
     prctl_get_bool(libc::PR_GET_THP_DISABLE)
 }
+
+/// Set an identifier (or reset it) to the address memory range.
+pub fn set_vma_anon_name(addr: NonNull<c_void>, length: NonZeroUsize, name: Option<&CStr>) -> Result<()> {
+    let nameref = match name {
+        Some(n) => n.as_ptr(),
+        _ => std::ptr::null_mut()
```

Comment:
Looks like we should use `null()` here

```suggestion
        _ => std::ptr::null()
```